src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -2867,3 +2867,29 @@ def test_python_unhandled_exception(self):
     self._validate_get_crash_data(data, expected_type, expected_address,
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
+                                  
+  def test_gdb_sigtrap(self):
+    """Test for GDB stack."""
+    data = self._read_test_data(
+        'gdb_sigtrap.txt')
+    expected_type = 'SIGTRAP'
+    expected_address = '0x000000000ac8'
+    expected_state = ('__aeabi_memcpy\nxymodem_trnasfer\nLoadImageFromUsb30\n')
```

Comment:
__aeabi_* should not in crash state since memcpy is not useful for deduplication, can you add that __aeabi_ to ignore list here - https://github.com/google/clusterfuzz/blob/master/src/python/crash_analysis/stack_parsing/stack_analyzer.py#L363